File: Frontend-Deploy-Center/src/types/Agent.ts
// src/types/Agent.ts
export interface Agent {
  id: number;
  name: string;
  ip: string;
  port: number;
  service_url: string;
  os: string;
  type: string;
  status: string;
  created_at?: string;
  updated_at?: string;
}

File: Backend-Deploy-Center/src/models/entity/agent.py
from pydantic import BaseModel
from typing import Optional
from datetime import datetime


class Agent(BaseModel):
    id: int
    name: str
    ip: str
    port: int
    service_url: Optional[str] = None
    os: str
    type: str
    status: str
    created_at: Optional[datetime] = None
    updated_at: Optional[datetime] = None